### PR TITLE
adding HashiCorp Vault namespaces examples to KMS doc

### PR DESCRIPTION
**Pull request by nicolaka**
#### Summary
I updated the KMS.md doc with additional details on using Vault Namespaces with cosign.

#### Ticket Link

Fixes

#### Release Note
NONE

```release-note

```

### CODE REVIEW

**Comment by cpanato** [comment]
@nicolaka please sign the DCO

**Review by hectorj2f — APPROVED** [review]

**Comment by nicolaka** [comment]
@cpanato done.

File: .github/workflows/scorecard_action.yml [hunk 1]
```diff
@@ -28,7 +28,7 @@ jobs:
           persist-credentials: false
 
       - name: "Run analysis"
-        uses: ossf/scorecard-action@c1aec4ac820532bab364f02a81873c555a0ba3a1
+        uses: ossf/scorecard-action@5c8bc69dc88b65c66584e07611df79d3579b0377
```
[the remark this answers is not in the document]
Comment (nicolaka) [reply .github/workflows/scorecard_action.yml:31]: not sure what happend there. I must of picked up this commit when i rebased following the instructions to sign a single commit. Shall I just open a new PR instead with DCO signed ?

[the remark this answers is not in the document]
Comment (cpanato) [reply .github/workflows/scorecard_action.yml:31]: rebase your pr and maybe remove the commit that you dont want

[the remark this answers is not in the document]
Comment (cpanato) [reply .github/workflows/scorecard_action.yml:31]: but feel free to close this and open a new one

**Pull request closed without merging**